cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -48,4 +48,6 @@
   public static final String RUN_BASE_COUNT_SIZE = "runBaseCountSize";
 
   public static final String RUN_BASE_COUNT_TIME = "runBaseCountTime";
+
+  public static final String ADAPTER_SPECIFICATION = "adapterSpeciciation";
```

Comment:
not needed. should change to ``ADAPTER_NAME``, after moving adapter name out of schedule name